sns/README.md

```diff
@@ -0,0 +1,25 @@
+# sns
+
+This library allows you to interact with the AWS SNS Service
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/sns
+```
+
+## Usage
```

Comment:
Can we add:
* An example of calling an inflight method
* A description saying what cloud resources are created when you compile to the AWS target, if any